src/components/SeedInfo/SeedInfoViews/FungalShifts.tsx

```diff
@@ -208,18 +224,46 @@ export const Shift: FC<IShiftProps> = props => {
         />
       </td>
       <td>
-        <OverlayTrigger placement="right" key="right" overlay={<Tooltip id={`tooltip-right`}>Shifted</Tooltip>}>
-          <Form.Check
-            checked={shifted}
-            onChange={e => {
-              setShifted(e.target.checked);
-            }}
-            type="checkbox"
-            id={`shifted`}
-            // label={`shifted`}
-            enterKeyHint="done"
-          />
-        </OverlayTrigger>
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            gap: "0.5rem",
+          }}
+        >
+          <OverlayTrigger placement="right" key="right" overlay={<Tooltip id={`tooltip-right`}>Shifted</Tooltip>}>
+            <Form.Check
+              checked={shifted}
+              onChange={handleSetShiftedClicked}
+              type="checkbox"
+              id={`shifted`}
+              // label={`shifted`}
+              enterKeyHint="done"
+            />
+          </OverlayTrigger>
+          {shifted && showTimer && (
+            <div
+              style={{
+                fontSize: "0.75rem",
+              }}
+            >
+              <CountdownCircleTimer
+                isPlaying
+                size={40}
+                strokeWidth={5}
+                duration={60}
```

Comment:
The duration here needs to be 300 (60 * 5) seconds, since the fungal shift timeout is 5 minutes